apps/hash-api/src/graph/ensure-system-graph-is-initialized/migrate-ontology-types/util.ts

```diff
@@ -317,9 +316,9 @@ export const createSystemPropertyTypeIfNotExists: ImpureGraphFunction<
   migrationState.propertyTypeVersions[baseUrl] = versionNumber;
 
   const existingPropertyType = await getPropertyTypeById(
-    context,
+    context.graphApi,
     authentication,
-    { propertyTypeId },
+    { propertyTypeId, temporalAxes: currentTimeInstantTemporalAxes },
   ).catch((error: Error) => {
```

Comment:
This is still expecting `getPropertyTypeById` to throw, but the new implementation returns `null` if not found

Reply:
Below we return the property type if it does exist. I think the `catch` can fully be removed?

Reply:
Yes, it can. All it's doing is converting to `null` if there's a `NotFoundError`, but the function isn't throwing that any more